docs/contributing.md

```diff
@@ -113,7 +113,7 @@ nox -s docs                    # Build and serve the documentation
 nox -s build                   # Make SDist and wheel
 ```
 
-More advanced users can run the update scripts. `update_pins` should work directly, but `update_constraints` needs all versions of Python installed. If you don't want to do that locally, a fast way to run it to use docker to run nox:
+More advanced users can run the update scripts. `update_pins` should work directly, but `update_constraints` needs all versions of Python installed. If you don't want to do that locally, a fast way to run it is to use docker to run nox:
```

Comment:
```suggestion
More advanced users can run the update scripts. `nox -s update_pins update_constraints`.
```

The `update_constraints` comment is most likely obsolete now that constraints are updated with `uv`.
The mention of `docker run ...` should be removed.